base/ltxref.dtx

```diff
@@ -87,6 +88,9 @@
 %  leaving a space on both sides of it is equivalent to leaving
 %  a space on either side.
 %
+%  Note: the starred version |\ref*| and |\pageref*| are provided
+%  to align with the of hyperref.
```

Comment:
Without \pkg{hyperref} (or some other package using the starred form) it is simply ignored.